readme.md

```diff
@@ -164,6 +164,12 @@ And don't forget to setup a cron job that runs daily on master, and include your
 
 Whenever you want to create a new release or when the extension is auto-published, the current date and time in the format `[year].[month].[date].[hour-minute]` (for example, `19.6.16.428`) is used as the version number. So that you don't have to manually update this number when a new release is to be made.
 
+#### Releases on Git tags
+
+You can also set up Travis to release extension on Git tags. In this case the version will be same as current Git tag. Automatic daily releases will not be triggered in this case.
```

Comment:
Can you mention how version names are affected by this change, as they will now be versioned based on Git tags value.

Reply:
I've written:

> In this case the version will be same as current Git tag.

Is this not OK? What should I do instead?

---

One thing that I forgot when I was developing this is that version will be exectly same as Git tag. This mean that it will also include prefix `v` (like `v1.2.3`). Is this OK or should I remove that prefix (so version will be just `1.2.3`)?

Reply:
> Is this not OK? What should I do instead?

Specifically link to "versioning" section, explaining something like everything mentioned in that section will be ignored if this environment variable is set.